src/app/typescript/v5/migrate/page.mdx

```diff
@@ -79,20 +148,20 @@ const onClick = async () => {
 | Deploy     | `sdk.deployer.deployBuiltInContract(...)`                | [`await deployPublishedContract(...)`](/references/typescript/v5/deploy/deployPublishedContract)              |
 
 
-### React Cheatsheet
+## React Cheatsheet
 
 | Task                 | `@thirdweb-dev/react`                                 | `thirdweb`                                          |
 | ---------------------| ----------------------------------------------------- | --------------------------------------------------- |
-| Provider             | `import { ThirdwebProvider} from @thirdweb-dev/react` | [`import { ThirdwebProvider } from "thirdweb/react"`](/react/v5/ThirdwebProvider) |
+| Provider             | `import { ThirdwebProvider} from @thirdweb-dev/react` | [`import { ThirdwebProvider } from "thirdweb/react"`](/typescript/v5/react/ThirdwebProvider) |
```

Comment:
this should be /react/v5/ThirdwebProvider (not sure why you changed it)

this whole cheat sheet should prob move to the react migration guide btw